pkg/visor/init.go

```diff
@@ -212,9 +212,22 @@ func initDmsg(ctx context.Context, v *Visor, log *logging.Logger) error {
 	}
 	dmsgC := dmsgc.New(v.conf.PK, v.conf.SK, v.ebc, v.conf.Dmsg)
 
-	time.Sleep(200 * time.Millisecond)
-	go dmsgC.Serve(context.Background())
-	time.Sleep(200 * time.Millisecond)
+	wg := new(sync.WaitGroup)
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		time.Sleep(200 * time.Millisecond)
```

Comment:
why the need for Sleep? is it for coordination or something else?

Reply:
It was already there. I Will check if it's still required or not and remove it if isn't.